### PR TITLE
fix: Handle empty project types

### PR DESCRIPTION
**Pull request by builditben**
Fix for #1699

### CODE REVIEW

File: api.planx.uk/saveAndReturn/utils.ts [hunk 1]
```diff
@@ -1,12 +1,10 @@
...
 import { format, addDays } from "date-fns";
 import { gql } from "graphql-request";
-import {
-  adminGraphQLClient as adminClient,
-} from "../hasura";
+import { adminGraphQLClient as adminClient } from "../hasura";
 import { LowCalSession, Team } from "../types";
 import { Template, getClientForTemplate, sendEmail } from "../notify/utils";
-import { _admin } from "../client";
+import { _admin as $admin } from "../client";
```
Comment (jessicamcinchak) [inline api.planx.uk/saveAndReturn/utils.ts:7]: mind sharing reasoning for this syntax change?

**Review by jessicamcinchak — APPROVED** [review]
Thanks for spotting & fixing this one - all emails sending on pizza before & after address and projectType have been set :100: 

Worth noting that I'm seeing "Project type not submitted" in each email I've received though, even when my session passport has `"proposal.projectType" = ["alter.repair"]` - but I'm guessing that is a query match issue unrelated to this fix?

File: api.planx.uk/saveAndReturn/utils.ts (hunk 1 above)
Comment (builditben) [reply api.planx.uk/saveAndReturn/utils.ts:7]: As @DafyddLlyr mentioned in the change that introduced `_admin` the underscore convention is generally used to indicate that a variable isn't referenced. I was working towards changing this shared planx-core client to `$admin` and `$public` instead if that works for everybody.

**Comment by builditben** [comment]
> Thanks for spotting & fixing this one - all emails sending on pizza before & after address and projectType have been set 100
> 
> Worth noting that I'm seeing "Project type not submitted" in each email I've received though, even when my session passport has `"proposal.projectType" = ["alter.repair"]` - but I'm guessing that is a query match issue unrelated to this fix we can investigate separately

@jessicamcinchak thanks for testing this.  The issue you mentioned might just be a data issue. I don't see any matches for `alter.repair` in the db: https://hasura.editor.planx.dev/console/data/default/schema/public/tables/project_types/browse?filter=value%3B%24eq%3Balter.repair

**Pull request merged**